packages/core/src/components.d.ts

```diff
@@ -8,6 +8,7 @@ import { HTMLStencilElement, JSXBase } from "@stencil/core/internal";
 export namespace Components {
     interface DsoAlert {
         "level": 'success' | 'info' | 'warning' | 'danger';
+        "status"?: string;
```

Comment:
Als ik in de implementatie kijk, dan lijkt het alsof `status` de tekstrepresentatie is van `level`, eigenlijk een `alt` voor als je `level` niet visueel tot je kunt nemen. Maar op basis van naamgeving in de interface zou ik dat nooit begrepen hebben.

Reply:
Goed punt. Zouden we die twee niet moeten koppelen? Voor elk `level` komt er een tekst te staan. Geen idee hoe dit component ingezet wordt overigens. @daverhelst / @gofix weten jullie dit? Of moeten we bij @bramhoosemans gaan shoppen voor info

Reply:
Nu ik de rest van het PR bekijk... Bij elk `level` wordt altijd dezelfde `status` gezet? Lijkt mij dat er dan een relatie is die wij prima in het Web Component kunnen onderbrengen

Reply:
Klopt, de sr-only tekst is afhankelijk van het "level". Maar hoe meer ik er naar kijken en jullie reacties lees, lijkt de naamgeving ook helemaal niet lekker op "level"-niveau. Naar mijn idee is het namelijk de "status" van de alert, en dan zou de sr-only tekst waarschijnlijk als "status-alt" genoemd kunnen worden. Wat vinden jullie?

Reply:
Inmiddels heb ik een andere oplossing bedacht, waarbij er een koppeling is tussen `level` en de bijbehorende sr-only tekst